src/components/CircularView.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useMemo } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+type Props = View['props'] & { size: number };
+
+export const CircularView = ({ size, style, ...otherProps }: Props) => {
+  const circleStyle = useMemo(() => getStyle(size).circle, [size]);
+
+  return <View {...otherProps} style={[circleStyle, style]} />;
+};
+
+const getStyle = (size: number) =>
+  StyleSheet.create({
+    // eslint does not detect the function being called at the top
+    // eslint-disable-next-line react-native/no-unused-styles
+    circle: {
+      alignItems: 'center',
+      aspectRatio: 1,
+      borderRadius: size / 2,
```

Comment:
could there be sizes, where `/ 2` will not result in circles here?
i think this should not depend on the size but always be something like 50%? is that possible, to always ensure a circle?

Reply:
why do you think this should not result in a circle for some `size` values?